YouTube Downloader.py

```diff
@@ -2,16 +2,16 @@
 import tkinter as tk
 from tkinter import *
 from pytube import YouTube
-import pyperclip as pc 
+import pyperclip as pc
 import validators as vd
+from PIL import ImageTk, Image
+from io import BytesIO
+import requests
```

Comment:
The requests can be imported way avove every other thing for code quality related issues :)
```suggestion
import requests

import tkinter as tk
from tkinter import *

```

Reply:
@Ashwin2105  Just address this issue and it is merged.